packages/consts/src/consts.ts

```diff
@@ -114,6 +119,19 @@ export const USERNAME = {
     REGEX: /^[a-zA-Z0-9_.-]{3,30}$/,
```

Comment:
You cannot form urls with protocol (like http://) because : and // are not allowed. But what about `www.`? Are we fine with that one? (Like you mentioned [here](https://github.com/apify/apify-shared-js/pull/486#discussion_r1827339816))

Reply:
Yeah, we agreed that it's not worth the hustle and would be too strict, as there are a lot of users / orgs named something.io, else.com, ...

It also means, that the apify-web won't be affected in any way.